coderd/workspaces.go

```diff
@@ -157,6 +159,10 @@ func (api *API) workspaces(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// To show the user's favorite workspaces first, we pass their userID and compare it to
+	// column favorite_of when ordering the rows.
+	filter.OrderByFavorite = apiKey.UserID
```

Comment:
review: this is part of how we ensure that 'favorite' workspaces only come up first for their respective owners.

Reply:
I think the naming is a bit confusing when read in query/code. I.e. "order by favorite" being the users ID.

How about something like `(Prioritize|Order)FavoritesForUser`?

Reply:
How about just `RequesterID`?